core/templates/pages/collection-editor-page/editor-tab/collection-node-creator.directive.ts

```diff
@@ -64,15 +64,13 @@ angular.module('oppia').directive('collectionNodeCreator', [
             if (isValidSearchQuery(searchQuery)) {
               ctrl.searchQueryHasError = false;
               return SearchExplorationsBackendApiService.fetchExplorations(
-                searchQuery
-              ).then(function(explorationMetadata) {
+                searchQuery).then(function(explorationMetadataList) {
```

Comment:
Add newline after `searchQuery`

Reply:
Won't it look weird?

Reply:
No -- it's common for the closing paren to be on the same level of indentation as the opening one, similar to how it was before.

Reply:
Ok, done